src/ListItems.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from "react";
+
+class ListItems extends Component {
```

Comment:
This is very much personal preference, but for very simple components like this is I'd definitely consider using functional components:
https://www.twilio.com/blog/react-choose-functional-components

Reply:
Ah ok thanks! I'll take a look at the link 🙂